### ISSUE TITLE
Flex component doesn't get display:flex style

### ISSUE THREAD

**Issue by iest** [petehunt/jsxstyle#21]
No matter what I try, the `<Flex>` component just doesn't ever have a `display` value of `flex`.

I dove into the source and in [autoprefix.js](https://github.com/petehunt/jsxstyle/blob/master/lib%2Fautoprefix.js#L87) the string `display:-webkit-flex;display:-ms-flexbox` is being added on to the style definition.

If you try:

```jsx
<div style={{display: 'flex;display:-webkit-flex;display:-ms-flexbox'}} />
```

the styles never come through. However:

```jsx
<div style={{display: 'flex'}} />
```

works as expected. Is this a react bug or what? Or am I being dumb and have missed something totally obvious?

**Comment by wmonk** [comment]
I saw your comment from twitter 😄 

Pulled the example directory and just changed it to use the `Flex` component. The CSS it generated works fine, because it puts it into a `<style>` tag like this:

```css
<style>
.jsxstyle3 {
width:192px;margin-left:auto;margin-right:auto;align-items:center;background:rgba(10, 0, 0,0.8);display:flex;display:-webkit-flex;display:-ms-flexbox;-webkit-align-items:center;
}
</style>
```

_not_ as an inline style.

**Comment by iest** [comment]
@wmonk that's using the jsxstyle webpack loader right? I'm using webpack usual style-loader... everything else works as expected — apart from the issue above.

**Comment by wmonk** [comment]
How does that work? The style loader should pickup imported stylesheets, whereas, as far as I can tell jsxstyle will just put everything into style tags. Can you post your webpack config?

**Comment by iest** [comment]
So I don't really know how or why, but in my project the styles are in-line on the dom nodes and jsxstyle isn't doing any appending to the head or anything.

Into the rabbit hole...

**Comment by iest** [comment]
Realised the last npm published version of this is `0.0.18` — installed direct from github and now everything works perfectly (with the styles being appended to the head).

So weird.
